src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -22,16 +26,39 @@ import { usePipelinerunActions } from './pipelinerun-actions';
 import { pipelineRunTableColumnClasses } from './PipelineRunListHeader';
 import { ScanStatus } from './ScanStatus';
 
+interface ReleasePlan {
+  spec: {
+    application: string;
+  };
+}
+
+interface Release {
+  spec: {
+    snapshot: string;
+  };
+}
+
 type PipelineRunListRowProps = RowFunctionArgs<
   PipelineRunKind,
   {
     vulnerabilities: { [key: string]: ScanResults };
     fetchedPipelineRuns: string[];
     error?: unknown;
+    releasePlan?: ReleasePlan;
+    release?: Release;
+    releaseName?: string;
```

Comment:
suggestion: maybe we could get the release name from the `release` itself? (`release.metadata.name`)